test/custom_task_test.go

```diff
@@ -1204,3 +1205,30 @@ func tearDownV1Beta1CustomTask(ctx context.Context, t *testing.T, c *clients, na
 
 	tearDown(ctx, t, c, namespace)
 }
+
+// updateConfigMap updates the config map for specified @name with values. We can't use the one from knativetest because
```

Comment:
was this moved from somewhere else or just a weird rebase?

Reply:
It used to be in the deleted `artifact_bucekt_test.go` and referenced here.